src/lib/VirtualScroller.scss

```diff
@@ -0,0 +1,16 @@
+.vs-container {
+    flex: 1 0 auto;
+    overflow-y: auto;
+
+    .vs-content-container {
```

Comment:
Try to avoid nesting in SCSS when possible. Keeping specificity low almost always makes things easier to maintain.